chapter_13_database_layer_validation.asciidoc

```diff
@@ -782,6 +761,8 @@ class ListViewTest(TestCase):
 //      looks like something that could be changed to a declarative sentence without noise like "can"
 //      maybe like "POST_on_existing_list_adds_item_to_the_list"?
 //      Also, from what I can see, other_list is not used at all in both tests. Is this intended?
+// CSANAD: not sure how it slipped my attention, true. We don't need the `other_list` in these
+// test cases and I might as well rename the test case
```

Comment:
i like explanatory variable names.  and i can't be bothered with the faff of feeding thru the commits for a test name change lol